crypto/keys/doc.go

```diff
@@ -0,0 +1,49 @@
+// nolint: golint
+
+/*
+	Package keys provides common key management API.
+
+
+	The Keybase interface
+
+	The Keybase interface defines the methods that a type needs to implement to be used
+	as key storage backend. This package provides few implementations out-of-the-box.
+
+	New
+
+	The New constructor returns an on-disk implementation backed by LevelDB storage that has been
+	the default implementation used by the SDK until v0.38.0. Due to security concerns, it is
+	recommended to drop it in favor of the NewKeyring or NewKeyringFile constructors as it will be
+	removed in future releases.
+
+	NewInMemory
+
+	The NewInMemory constructor returns an implementation backed by an in-memory, goroutine-safe
+	map that has historically been used for testing purposes or on-the-fly key generation as the
+	generated keys are discarded when the process terminates or the type instance is garbage
+	collected.
+
+	NewKeyring
+
+	The NewKeyring constructor returns an implementation backed by a keyring library
+	(https://github.com/99designs/keyring), whose aim is to provide a common abstraction and uniform
+	interface between secret stores available for Windows, macOS, and most GNU/Linux distributions
+	as well as operating system-agnostic encrypted file-based backends.
+
+	The backends:
+		os		The instance returned by this constructor uses the operating system's default
```

Comment:
Formatting seems off here.

Reply:
Tested with `go doc`

Reply:
Right, but it seems off. The `os` doesn't line up with the other entries.

Reply:
Oh I see why. I used tabs for formatting (same as https://golang.org/src/fmt/doc.go). Rendered result seems correctly aligned though (see below)